upgrades/upgrade_testing_framework/steps/step_start_target_cluster.py

```diff
@@ -0,0 +1,31 @@
+from upgrade_testing_framework.core.framework_step import FrameworkStep
+from upgrade_testing_framework.cluster_management.cluster import Cluster, ClusterNotStartedInTimeException
+
+class StartTargetCluster(FrameworkStep):
```

Comment:
Discussion question:
Ideally do we want `StartSouceCluster` and `StartTargetCluster` steps to be totally different, or should we just have a `StartCluster` step that takes in a config? 
Changing this would be out of scope for this review. My first reaction is to object to the code duplication, but when I think about it more, source and target are pretty fundamental concepts for a migration/upgrade, and there's not (at this point) any reason to have more than the two, so maybe it's not a problem.

Reply:
Yeah, I had the same thought about making a base step for this, having them share library code, make a meta class, etc.  I ended up deciding it wasn't worth messing with now as there's only two of them.  Once we hit three, or it becomes a hassle to maintain the separate "copies", then we can revisit it.  There are bigger fish to fry at the moment.